src/messages/welcome.json

```diff
@@ -1,3 +1,3 @@
 {
-    "message": "<p>Welcome!</p> <p>If you need help at any time, press the H key. To continue, select the start button below.</p>"
+    "message": "<p>Welcome!</p> <p>This tool will guide you through setting up your computer the way you like it best.</p> <p>For help at any time, press the H key. To continue, select the next button or press the N key.</p>"
```

Comment:
Checked with Dana in the channel, she suggests we keep it anyway, "so folks understand that we will have that functionality eventually"